package.json

```diff
@@ -22,16 +22,18 @@
   "types": "dist/FastMCP.d.ts",
   "dependencies": {
     "@modelcontextprotocol/sdk": "^1.6.0",
+    "@standard-schema/spec": "^1.0.0",
     "execa": "^9.5.2",
     "file-type": "^20.3.0",
     "fuse.js": "^7.1.0",
     "mcp-proxy": "^2.10.4",
     "strict-event-emitter-types": "^2.0.0",
     "undici": "^7.4.0",
     "uri-templates": "^0.2.0",
+    "xsschema": "0.2.0-beta.2",
     "yargs": "^17.7.2",
     "zod": "^3.24.2",
-    "zod-to-json-schema": "^3.24.3"
+    "zod-to-json-schema": "^3.24.5"
```

Comment:
This library is a peer dependency of xsschema and isn't strictly required anymore but I left it because removing it would be a breaking change for Zod users.

Reply:
I would remove it. The whole idea behind all of this is not to add a new dependency all-together.
And it won't break in runtime but at the install moment. Seems safe enough.